Parse/ParseClientConfiguration.h

```diff
@@ -10,6 +10,7 @@
 #import <Foundation/Foundation.h>
 
 #import <Parse/PFConstants.h>
+#import "PFFileUploadController.h"
```

Comment:
Ok, done, it builds, but I have to commit to test it. Here goes... 

I've also added `#import <Parse/PFFileUploadController.h>` to the global import header.

Out of curiosity, why prefer the forward @protocol over the import? Don't see any cycles here.